ansible/roles/host-ocp4-hcp-cnv-destroy/tasks/main.yaml

```
@@ -0,0 +1,27 @@
+---
+- name: Log in (obtain access token)
+  community.okd.openshift_auth:
+    username: "{{ sandbox_openshift_username }}"
+    password: "{{ sandbox_openshift_password }}"
+    host: "{{ sandbox_openshift_api_url }}"
+  register: k8s_auth_results
+  when: sandbox_openshift_username | default(false)
+
+- name: Install OCP using HCP
+  module_defaults:
+    group/k8s:
+      host: "{{ sandbox_openshift_api_url }}"
+      api_key: "{{ k8s_auth_results.k8s_auth.api_key|default(sandbox_openshift_api_key) }}"
+      validate_certs: false
+
+  block:
+    - name: Remove the HostedCluster
+      ignore_errors: True
```

true

Also if we ignore errors should be try a few times? Like 3 retries?